NuGetPackageAnalyzerFunction/NuGetPackageAnalyzerFunction.csproj

```diff
@@ -0,0 +1,26 @@
+<Project Sdk="Microsoft.NET.Sdk">
+	<PropertyGroup>
+		<TargetFramework>netcoreapp3.1</TargetFramework>
+		<AzureFunctionsVersion>v3</AzureFunctionsVersion>
+		<UserSecretsId>2d2c1cd1-44a9-45c1-ba2b-155d2d30e78e</UserSecretsId>
+	</PropertyGroup>
+	<ItemGroup>
+		<PackageReference Include="Microsoft.Azure.WebJobs.Extensions.ServiceBus" Version="4.1.0" />
+		<PackageReference Include="Microsoft.NET.Sdk.Functions" Version="3.0.3" />
+		<PackageReference Include="Microsoft.ApplicationInsights" Version="2.8.1" />
+		<PackageReference Include="Microsoft.Azure.WebJobs.ServiceBus" Version="3.0.0-beta8" />
+		<PackageReference Include="System.Data.SqlClient" Version="4.8.1" />
+	</ItemGroup>
+	<ItemGroup>
+		<None Update="host.json">
+			<CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+		</None>
+		<None Update="local.settings.json">
+			<CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+			<CopyToPublishDirectory>Never</CopyToPublishDirectory>
+		</None>
+	</ItemGroup>
+	<ItemGroup>
+	  <ProjectReference Include="..\..\NuGet.Client\src\NuGet.Core\NuGet.Packaging\NuGet.Packaging.csproj" />
```

Comment:
Can this use `PackageReference` instead?

Reply:
Thanks for reviewing! But sorry, I can't.  
All the Azure function could only targets netcoreapp3.1 for now (as net5.0 is not GA yet).
And the NuGet doesn't support signing in netcoreapp3.1. So there is no NuGet.Packaging package available for signing related APIs for TFM netcoreapp3.1.
I have to manually change the NuGet.Client solution, to retarget it to netcoreapp3.1 and make the verification APIs  available for netcoreapp3.1 code path. 
Fortunately, the netcoreapp3.1 runtime has no difference with net5.0 in the API I'm going to use. (I remember netcroreapp3.1 runtime doesn't support signing well, but the verification works the same way with net5.0)

But if the analyzing is not relying on net5.0 runtime, we can use package reference instead. So the analyzer in this PR is a very special case.

Reply:
Makes sense, thanks for the info!